apollo-assembly/src/main/resources/application.yml

```diff
@@ -33,3 +33,5 @@ eureka:
     healthcheck:
       enabled: true
     eurekaServiceUrlPollIntervalSeconds: 60
+    fetch-registry: false
```

Comment:
It seems apollo-adminservice would not register itself in apollo-assembly mode. Maybe ConfigServerEurekaClientConfigure should be applied to both apollo-configservice and apollo-adminservice?

Reply:
Yes, I missed the adminservice in apollo-assembly mode